src/argilla/client/feedback/dataset/local/dataset.py

```diff
@@ -11,16 +11,43 @@
 #  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
-
+import logging
+import textwrap
```

Comment:
Minor formatting nit

```suggestion

import logging
import textwrap
```